File: 7.py
# -*- coding:utf-8 -*-
# 题目: 斐波拉契数列


class Solution:
    def Fibonacci(self, n):
        # write code here
        firstNum = 0
        secondNum = 1
        index = 0
        while index < n:
            firstNum, secondNum = secondNum, firstNum + secondNum
            index += 1
        return firstNum


obj = Solution()
print(obj.Fibonacci(6))

File: 0-9/7.go
package main

import "fmt"

// 题目: 斐波拉契数列

// Fibonacci ...
/**
 *
 * @param n int整型
 * @return int整型
 */
func Fibonacci(n int) int {
	// write code here
	if n == 0 {
		return 0
	}
	numOne := 0
	numTwo := 1
	for i := 0; i < n; i++ {
		numOne, numTwo = numTwo, numOne+numTwo
	}

	return numOne
}

func main() {
	fmt.Println(Fibonacci(4))
}
